Packs/PATHelpdeskAdvanced/ReleaseNotes/1_0_3.md

```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### PAT HelpdeskAdvanced
+- Updated the integration to support a newer API version.
```

Comment:
lets document this in the readme *this integration is tested with version x.y.z* like we have in others